pipelines/scipipe/ap_verify.groovy

```diff
@@ -438,16 +438,32 @@ def void runApVerify(Map p) {
 
       cd ${RUN_DIR}
       # -p ignored if not -g 3
+      status=0
+      extra_args=""
+      # deal with NAMESPACE and REST_PROXY_URL being empty strings
+      if [[ -n "$NAMESPACE" || -n "$REST_PROXY_URL" ]]; then
+        [[ -n "$NAMESPACE" ]] && extra_args+=" -n ${NAMESPACE}"
+        [[ -n "$REST_PROXY_URL" ]] && extra_args+=" -u ${REST_PROXY_URL}"
+      fi
       run_ci_dataset.sh -d ${DATASET_NAME} \
                         -g ${DATASET_GEN} \
                         -p ${DATASET_PIPE} \
-                        -n ${NAMESPACE} \
-                        -u ${REST_PROXY_URL} \
+                        "${extra_args} " \
                         -e ci_id=$BUILD_ID \
                         -e ci_url=$BUILD_URL \
                         -e ci_name=$JOB_NAME \
                         -e ci_refs=$JOB_REFS \
-                        -e pipeline=$JOB_PIPELINE
+                        -e pipeline=$JOB_PIPELINE || status=$?
+      # add backwards compatibility - to be removed after release v29
```

Comment:
I am not so sure the backwards compatibility is needed
https://rubin-ci-dev.slac.stanford.edu/blue/organizations/jenkins/scipipe%2Fap_verify/detail/ap_verify/24/pipeline/68

The `v29.1.0.rc2` tag works without, the problem before was the zero string arguments

Reply:
It looks like the "backwards compatibility" is removing all the `-e` metadata? That was added in `w.2025.12`.

In any case, I agree that it's probably better not to have any temporary patches, given this code's tendency to bit-rot. Even if it didn't work for `v29`, it would probably be cleaner to just let it fail one last time.